launcher/minecraft/launch/LauncherPartLaunch.cpp

```diff
@@ -159,6 +160,74 @@ void LauncherPartLaunch::executeTask()
 
     qDebug() << args.join(' ');
 
+    args.prepend(FS::ResolveExecutable(instance->settings()->get("JavaPath").toString()));
+
+#ifdef Q_OS_LINUX
+    // TODO: Sandboxing setting
+    if (true) {
+        QString actualRuntimeDir = QStandardPaths::writableLocation(QStandardPaths::RuntimeLocation);
+        QString sandboxedRuntimeDir = "/tmp";
+
+        QStringList bwrapArgs{};
+        bwrapArgs << QStandardPaths::findExecutable(BuildConfig.LINUX_BWRAP_BINARY);
+        bwrapArgs << "--unshare-user" << "--unshare-ipc" << "--unshare-pid" << "--unshare-uts" << "--unshare-cgroup";
+        bwrapArgs << "--share-net";
+        bwrapArgs << "--die-with-parent";
+        bwrapArgs << "--unsetenv" << "DBUS_SESSION_BUS_ADDRESS";
+
+        // default binds
+        bwrapArgs << "--dev" << "/dev";
+        bwrapArgs << "--dev-bind-try" << "/dev/dri" << "/dev/dri";
+        bwrapArgs << "--proc" << "/proc";
+        bwrapArgs << "--tmpfs" << "/tmp";
+        bwrapArgs << "--ro-bind" << "/etc" << "/etc";
+        bwrapArgs << "--ro-bind" << "/usr" << "/usr";
+        bwrapArgs << "--ro-bind-try" << "/sys/class" << "/sys/class";
+        bwrapArgs << "--ro-bind-try" << "/sys/dev/char" << "/sys/dev/char";
+        bwrapArgs << "--ro-bind-try" << "/sys/devices/pci0000:00" << "/sys/devices/pci0000:00";
+        bwrapArgs << "--ro-bind-try" << "/sys/devices/system/cpu" << "/sys/devices/system/cpu";
+        bwrapArgs << "--ro-bind-try" << "/run/systemd/resolve" << "/run/systemd/resolve";
```

Comment:
This seems distro specific. I'm not sure what is needed for resolvers other than systemd-resolved.

Reply:
This is indeed distro specific. But I think it's a good default, especially as it's `--ro-bind-try`